src/config/navigation/tabConfig.tsx

```diff
@@ -102,3 +106,31 @@ export const tabNavigatorConfig: TabNavigatorConfig = {
     aboutTabConfig
   ]
 };
+
+export const createDynamicTabConfig = (
+  accessibilityLabel: string,
+  iconName: string,
```

Comment:
should we use a question mark icon as default icon, if nothing is set?

Reply:
Instead of specifying a standard icon, I find it better to integrate `Icon.NamedIcon` into the `OrientationAwareIcon` component. This way we can use Tabler icons as we do in `HomeService`. Currently we can only use the icons that are attached to the `Icon` component.

Reply:
Yes, being able to use icons the way we can use it for service tiles is a good idea.